cluster/gce/config-default.sh

```diff
@@ -36,6 +36,11 @@ NODE_IMAGE_PROJECT=${KUBE_GCE_NODE_PROJECT:-"${MASTER_IMAGE_PROJECT}"}
 CONTAINER_RUNTIME=${KUBE_CONTAINER_RUNTIME:-docker}
 RKT_VERSION=${KUBE_RKT_VERSION:-0.5.5}
 
+# If the distro is CoreOS, kubelet will run as a systemd service.
+if [[ "${OS_DISTRIBUTION}" == "coreos" ]]; then
```

Comment:
Looking at https://github.com/kubernetes/kubernetes/blob/master/cluster/saltbase/pillar/systemd.sls, I worry this is a bit too fragile.

Can we instead somehow detect at run-time in the code whether the kubelet is running under systemd or not?